src/frontend/post-github-configuration.ts

```diff
@@ -52,12 +53,15 @@ export default async (req: Request, res: Response): Promise<void> => {
 			}
 		}
 
-		await Subscription.install({
+		const subscription = await Subscription.install({
 			clientKey: getHashedKey(req.body.clientKey),
 			installationId: req.body.installationId,
 			host: res.locals.jiraHost
 		});
 
+		await findOrStartSync(subscription);
```

Comment:
Moved the logic of starting the sync outside the install function. This is to remove the cyclic dependency. Subscription class should be merely for data manipulation, and sync should be started externally.